protocol/avdl/chat1/local.avdl

```diff
@@ -256,8 +258,23 @@ protocol local {
     string senderUsername;
     string senderDeviceName;
     string senderDeviceType;
+
+    // MessageBoxed.V1: Hash of the encrypted header ciphertext.
+    // MessageBoxed.V2: Hash of MessageBoxed.headerSealed (.v || .n || .b)
```

Comment:
This makes me happy. (We'll need to specify how the version int is serialized. I arbitrarily suggest an unsigned 64 bit int in network byte order.)